wp-content/themes/engage/templates/single.twig

```diff
@@ -3,8 +3,11 @@
 {% block content %}
     <div class="post__wrapper wrapper--pad">
         <div class="article__wrapper container container--lg">
-            {% set showFeaturedImage = post.thumbnail.src and post.post_type != 'team' and post.post_type != 'board' %}
-            <article class="article post-type-{{ post.post_type }} {{ showFeaturedImage ? 'article--has-featured-img' }}" id="post-{{ post.ID }}">
+            {% set showVideo = post.getVideo and post.post_type != 'team' and post.post_type != 'board' %}
+            {% if not showVideo %}
+              {% set showFeaturedImage = post.thumbnail.src and post.post_type != 'team' and post.post_type != 'board' %}
```

Comment:
I'd remove this if statement around setting the variable. Generally variables get set, even if they're not going to be used in this flow (ie because an if statement above will not get to it). A better design here, IMO, would be to change `show` in `showVideo` and `showFeaturedImage` to `has`. The `show` there isn't really what you're deciding. You're seeing if we _have_ them, then deciding what to show later.